tests/lib/types/types.test.ts

```diff
@@ -27,6 +27,7 @@
 
 import { AST, ESLint, Linter, loadESLint, Rule, RuleTester, Scope, SourceCode } from "eslint";
 import { ESLintRules } from "eslint/rules";
+import { Linter as ESLinter } from "eslint/universal";
```

Comment:
Shall we add a line with `new ESLinter()` to the tests? This would have caught the error in https://github.com/eslint/eslint/pull/18883#discussion_r1758884479.

Reply:
I think that's a good idea. :1:

Reply:
https://github.com/eslint/eslint/pull/18883/commits/11d44574d03ba65a0403d2e3b20d34dc6b291752